exercises/concept/lucky-numbers/.docs/instructions.md

```diff
@@ -1,7 +1,7 @@
 # Instructions
 
 Your friend Kojo is a big fan of numbers.
-He has a small website for playing around with numbers: www.fun-with-numbers.com.
+He has a small website for playing around with numbers: www.kojo-presents-fun-with-numbers.com.
```

Comment:
Why would we have a link that goes nowhere?  It is not guaranteed to go nowhere and may need to be changed later.